src/Traversals/depthfirst.jl

```diff
@@ -142,3 +142,52 @@ end
 end
 
 @traitfn is_cyclic(g::::(!IsDirected), alg::DepthFirst=DepthFirst()) = ne(g) > 0
+
+############ toposort2
+#
+mutable struct TopoSortState2{T<:Integer} <: AbstractTraversalState
+    vcolor::Vector{UInt8}
```

Comment:
Don't we have this new Bivector for that? Or is is not ready yet?

Reply:
We do, but that's a separate PR that needs some benchmarking. But yes - this sort of thing is exactly why I wrote `DiBitVectors`.